tellmewhy-sir/code-challenge-datamade
# Code challenge solution

## Overview

Provides a code solution for the challenge that addresses all the TODOs

### Notes

I had to edit the dockerfile because it was failing to build saying npm couldn't be found.

## Testing Instructions

- Run the `docker-compose` commands to start the app
- Open localhost:8000 to view the app
- Enter a valid address and confirm the results returned from API
